test/e2e/specs/specs-playwright/wp-blocks__coblocks-gutter-spec.ts

```diff
@@ -17,13 +18,22 @@ describe( DataHelper.createSuiteTitle( 'WPCOM-specific gutter controls' ), () =>
 	let pricingTableBlock: PricingTableBlock;
 	let page: Page;
 
+	let user: string;
+	if ( BrowserHelper.targetCoBlocksEdge() ) {
```

Comment:
Nit, and no need to do now, but: could we move this logic to the helper above and abstract into a `BrowserHelper.getUser` or something along those lines?

Reply:
Yeah, I've had my eye on that change too - we should do this also for our gutenberg edge specific tests.

We are going to move some library things around soon where we move all ENV variable reading to its own file, and then add some user-related helpers to BrowserHelper that can be re-used across different specs!